src/core/Do.Architecture/Architecture/IFeature.cs

```diff
@@ -2,5 +2,6 @@ namespace Do.Architecture;
 
 public interface IFeature
 {
+    string Id => GetType().Name;
```

Comment:
leave it as `string Id { get; }`, do not introduce a default implementation here